app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name = command.split()
+    if (name[0] == "cp"
+            and name[1] != name[2]):
+        with open(name[1], "r") as origin:
+            content = origin.read()
+        with open(name[2], "w") as copy:
+            copy.write(content)
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220112792-385c0799-d6e1-4704-bd62-e9d0146df3f3.png)
no need to use with keyword for 2 times here